apps/src/storage/dataBrowser/LibraryTable.jsx

```diff
@@ -135,7 +143,8 @@ class LibraryTable extends React.Component {
 
 export default connect(
   state => ({
-    libraryManifest: state.data.libraryManifest || {}
+    libraryManifest: state.data.libraryManifest || {},
+    locale: state.pageConstants && state.pageConstants.locale
```

Comment:
Is this always set? Is there a fallback in case it isn't?

Reply:
I believe it's always set, but even if it's not, moment will just default to english if no locale is set, so we should be fine either way.